packages/core/strapi/lib/commands/actions/ts/generate-types/command.js

```diff
@@ -10,12 +10,7 @@ module.exports = ({ command }) => {
   command
     .command('ts:generate-types')
     .description(`Generate TypeScript typings for your schemas`)
-    .option(
-      '-o, --out-dir <outDir>',
-      'Specify a relative directory in which the schemas definitions will be generated'
-    )
-    .option('-f, --file <file>', 'Specify a filename to store the schemas definitions')
-    .option('--verbose', `Display more information about the types generation`, false)
```

Comment:
We may want to add a "deprecation" to this and silently enable `debug` when it's used just so we don't break anything

Reply:
Yeah, I was wondering the same. We could do that indeed

Reply:
[`489e422` (#16757)](https://github.com/strapi/strapi/pull/16757/commits/489e422f60fac25b346e270b8eeab5e4c3c19e46)